### PR TITLE
Added files and added comments

### PR DESCRIPTION
**Pull request by CS50X-RGB**
Fixes  issue  number #9

### CODE REVIEW

**Comment by CS50X-RGB** [comment]
@kodeflap plz review my pr

**Comment by kodeflap** [comment]
I will review your PR

File: app/src/main/java/com/dlight/algoguide/dsa/searching/linear_search/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# WHAT IS LINEAR SEARCH
```
Comment (kodeflap) [inline app/src/main/java/com/dlight/algoguide/dsa/searching/linear_search/README.md:1]: There is no need for this readme file

File: app/src/main/java/com/dlight/algoguide/dsa/sorting/bubble_sort/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## WHAT IS BUBBLE SORT
```
Comment (kodeflap) [inline app/src/main/java/com/dlight/algoguide/dsa/sorting/bubble_sort/README.md:1]: This readme is not required

File: app/src/main/java/com/dlight/algoguide/dsa/sorting/insertion_sort/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# WHAT IS INSERTATION SORT??
```
Comment (kodeflap) [inline app/src/main/java/com/dlight/algoguide/dsa/sorting/insertion_sort/README.md:1]: Readme file not required here

**Review by kodeflap — CHANGES_REQUESTED** [review]
Thanks for the improved code. The suggestion is that there is no need for additional readme files for each sort in here. So you can remove it and can make other PR.

**Comment by CS50X-RGB** [comment]
Ok I will fix it

**Comment by CS50X-RGB** [comment]
Hi @kodeflap how can I make this this under the same pr??

**Comment by CS50X-RGB** [comment]
@kodeflap i did the changes

**Review by kodeflap — APPROVED** [review]
your PR request will be merged

**Pull request merged**